src/globus_sdk/services/flows/client.py

```diff
@@ -214,7 +222,7 @@ def list_flows(
         :param filter_fulltext: A string to use in a full-text search to filter results
         :type filter_fulltext: str, optional
         :param orderby: A criterion for ordering flows in the listing
-        :type orderby: str, optional
+        :type orderby: str or tuple, optional
```

Comment:
Can you add a piece to the docstring here indicating what the required string format is (since we still support it in addition to the tuple)?